src/site/stages/build/process-cms-exports/schemas/raw/paragraph-situation_update.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+	type: 'object',
+	properties: {
+		field_date_and_time: {
+			type: 'array',
+			items: {
+				properties: {
+					// These are actually timestamps like: 2019-05-30T21:00:00
```

Comment:
I actually just ran into one today that actually includes the timezone. `2019-05-30T21:00:00-04:00`. Is that true of these as well?